js/script.js

```diff
@@ -11,83 +11,83 @@
 //When user inputs text into textbox, clicking enter will submit it
 var inputByEnter = document.getElementById("textbox");
 
-inputByEnter.addEventListener("keyup", function(event){
-	//13 signifies the Enter key
-	if(event.keyCode === 13){	
-		//This cancels the default action if necessary
-		event.preventDefault();
-		//Call the addItem() function below
-		addItem();
-	}
+inputByEnter.addEventListener("keyup", function (event) {
+  //13 signifies the Enter key
+  if (event.keyCode === 13) {
+    //This cancels the default action if necessary
+    event.preventDefault();
+    //Call the addItem() function below
+    addItem();
+  }
 });
 
 
 //Create a new item on the list
-function addItem() {
+function addItem () {
 
-	//Create new list element
-	var li = document.createElement("li");
+  //Create new list element
+  var li = document.createElement("li");
 
-	//Set a 'contentEditable' attribute to keep track of whether checkbox is checked
-	li.setAttribute('contentEditable', 'false');
+  //Set a 'contentEditable' attribute to keep track of whether checkbox is checked
+  li.setAttribute('contentEditable', 'false');
```

Comment:
When using attributes or setting attributes, make sure to read what this attribute will actually do